.gitignore

```diff
@@ -134,7 +134,9 @@ dmypy.json
 
 # Specific Excel files
 Round 2 Reporting - Consolidation.xlsx
+Round 2 Reporting - Consolidation (MASTER).xlsx
```

Comment:
you could just use the * wildcard to exclude any file that begins with "Round 2" or something here?